### PR TITLE
Add Blocklist and deprecated Blacklist

### PR DESCRIPTION
**Pull request by dara123**

### CODE REVIEW

**Review by dortort — DISMISSED** [review]

**Review by slavcodev — COMMENTED** [review]
Please update CHANGELOG

File: CHANGELOG.md [hunk 1]
```diff
@@ -13,6 +13,14 @@ Fixed - for any bug fixes.
 Security - in case of vulnerabilities.
 -->
 
+## [2.9.0] 2020-08-03
```
Comment (slavcodev) [inline CHANGELOG.md:16]: It should be in `Unreleased`

Comment (slavcodev) [reply CHANGELOG.md:16]: Or if you gonna release than you also need to update `Client` constant

Comment (dara123) [reply CHANGELOG.md]: thanks. got it 👍

File: CHANGELOG.md [hunk 2]
```diff
@@ -15,6 +15,12 @@ Security - in case of vulnerabilities.
 
 ## [Unreleased]
 
+### Added
+- [x] Added Blocklist
```
Comment (slavcodev) [inline CHANGELOG.md:19]:
```suggestion
- [x] Added *Blocklist API*
```

**Review by slavcodev — DISMISSED** [review]

File: CHANGELOG.md [hunk 3]
```diff
@@ -15,6 +15,12 @@ Security - in case of vulnerabilities.
 
 ## [Unreleased]
 
+### Added
+- [x] Added *Blocklist API*
```
Comment (slavcodev) [inline CHANGELOG.md:19]:
```suggestion
- [x] Added **Blocklist API**
```

**Review by slavcodev — APPROVED** [review]

**Review by ashkarpetin — APPROVED** [review]

**Pull request merged**